### ISSUE TITLE
Requirements module stubs/interface

### ISSUE THREAD

**Issue by ianrowan** [hicommonwealth/commonwealth#4958]
## Description
Set up method stubs/interface for the requirements module
- Validate requirments method
- Check group membership per address method

## Engineering Requirements
- Stubs should exist that will be used for implementation in future tickets. Stubs should be sufficient to unblock Route work
- Stub based unit test created 
- module file structure setup/locked in 

## Acceptance Criteria

N/A 

## Additional Context
Breaking this into a separate ticket from logic to allow for merge of stubs for use by route work
